src/main/java/command/ViewQuotaCommand.java

```diff
@@ -0,0 +1,48 @@
+package command;
+
+import exceptions.SEPException;
+import exceptions.SEPFormatException;
+import exceptions.SEPNotFoundException;
+
+import studentlist.StudentList;
+import ui.UI;
+import university.University;
+import university.UniversityRepository;
+
+public class ViewQuotaCommand extends Command {
+    private String input;
+    private UI ui;
+
+    public ViewQuotaCommand(StudentList studentList, String input, UI ui) {
+        super(studentList);
+        this.input = input;
+        this.ui = ui;
+    }
+
+    @Override
+    public void run() {
+        try {
+            String[] parts = input.split(" ");
+            if (parts.length != 2) {
+                throw SEPFormatException.rejectViewQuotaFormat();
+            }
+
+            String uniIndeString = parts[1].trim();
+            if (!uniIndeString.matches("-?\\d+")) {
```

Comment:
Perhaps it's better to factor out "-?\\d+" as a constant instead?

Reply:
This is a good point